src/QPMotionConstr.cpp

```diff
@@ -270,8 +295,9 @@ void MotionConstr::update(const std::vector<rbd::MultiBody> & mbs,
 {
   computeMatrix(mbs, mbcs);
 
-  AL_.head(torqueL_.rows()) += torqueL_;
-  AU_.head(torqueU_.rows()) += torqueU_;
+  // max[tauMin, tauDMin*dt + tau(k-1)] - C <= H*alphaD - J^t G lambda <= min[tauMax, tauDMax * dt + tau(k-1)] - C
+  AL_.head(torqueL_.rows()) += torqueL_.cwiseMax(torqueDtL_ + lastTorque_);
+  AU_.head(torqueL_.rows()) += torqueU_.cwiseMin(torqueDtU_ + lastTorque_);
```

Comment:
So on the first iteration this is not exaclty correct but there's also a good chance that `torqueDtL_ < torqueL_` and `torqueDtU_ > torqueU_` (at least it is true on the Panda where we want/need to implement this) so this is probably fine